docs/semantics.md

```diff
@@ -199,22 +200,22 @@ Even though A/, A/B/, and C/ are directories in the filesystem, a 0-byte object
 
 The above example was based on greenfield deployments which assumes starting fresh, where the directories are created from Cloud Storage FUSE. If a user unmounts this Cloud Storage FUSE bucket, and then re-mounts it to a different path, the user will see the directory structure correctly in the filesystem because it was originally created by Cloud Storage FUSE.
 
-However, If a user already has objects with prefixes to simulate a directory structure in their buckets that did not originate from Cloud Storage FUSE, and mounts the bucket using Cloud Storage FUSE, the directories and objects under the directories will not be visible until a user manually creates the directory using mkdir on the local instance. This is because with Cloud Storage FUSE, directories are by default not implicitly defined; they exist only if a matching object ending in a slash exists.
-
+However, if a user already has objects with prefixes to simulate a directory structure in their buckets that did not originate from Cloud Storage FUSE, and mounts the bucket using Cloud Storage FUSE, the directories and objects under the directories will not be visible until a user manually creates the directory, with the same name, using mkdir on the local instance (or by creating the folder in the WebUI). This is because with Cloud Storage FUSE, directories are by default not implicitly defined; they exist only if a matching object ending in a slash exists.  These backing objects for directories are special 0-byte objects, which are placeholders for directories and are not are not created by the Cloud Storage CLI tools such as gcloud or gsutil.
 
-So if a user has the following objects in their Cloud Storage buckets, that were not originally created via Cloud Storage FUSE (for example created by uploading a local directory using ``` gsutil cp -r``` command)
+So if a user has the following objects in their Cloud Storage buckets, that were not originally created via Cloud Storage FUSE (for example created by uploading a local directory using `gsutil cp -r` command).
+- A/1.txt
+- A/B/2.txt
+- C/3.txt
 
+then mounting the bucket and running ```ls``` to see its content will not show any files until the directories A/, A/B/, and C/ are created on the local filesystem using the `mkdir` command (or create these from the WebUI). A user would have to run `mkdir ./A`, `mkdir ./A/B` and `mkdir ./C`.
```

Comment:
are created on the local filesystem using the `mkdir` command (or create these from the WebUI), can remove are (or create these from the WebUI) from this text as webUI will show A, B and C as folders, it might confuse the reader.

Reply:
Wasn't the motive of updating this document to highlight that there are methods other than `mkdir` for creating directories?

Reply:
You removed `or create these from the WebUI`. Is this intended?

Reply:
Yes, I agree with @ankitaluthra1 here that this will cause confusion as implicit directories will already be visible on WebUI. Also, since we have already mentioned how simulated folders can be created in the previous paragraph, it is redundant. What do you think? CC: @marcoa6

Reply:
Umm, agree that we already convey how simulated folders are created.